frontend/src/components/node/Details.tsx

```diff
@@ -45,6 +53,11 @@ export default function NodeDetails() {
   const [node, setNode] = useState(nodeFromAPI);
   const noMetrics = metricsError?.status === 404;
 
+  const [drainDialogOpen, setDrainDialogOpen] = useState(false);
+  const [cordonDialogOpen, setCordonDialogOpen] = useState(false);
```

Comment:
According to #1316 we only need the confirmation dialog to the drain action, not the cordon.

Reply:
Done!